docs/source/benchmarking/materials_science_and_engineering/oiemanufacturing.rst

```diff
@@ -1,12 +1,12 @@
 Open Innovation Environment (OIE) domain ontologies, Manufacturing module (OIEManufacturing)
```

Comment:
convert `Open Innovation Environment (OIE) domain ontologies, Manufacturing module (OIEManufacturing)` to `Open Innovation Environment Manufacturing (OIEManufacturing)`